### PR TITLE
Create Japanese Translation documents for Workshop

### PR DESCRIPTION
**Pull request by yoshioterada**
Create Japanese Translation documents for Workshop

### CODE REVIEW

**Comment by dingmeng-xue** [comment]
@yoshioterada , thanks for contribution. Could you add somebody who can read Japanese for review? Then, we can merge it.

**Comment by m-moris** [comment]
@yoshioterada 

Thanks for writing the Japanese documentation of the detailed procedure.
I attended this workshop and reviewed these documents.

**Comment by suzukiyuhsuke** [comment]
@yoshioterada,
Sorry my late response, I'd think that chapter-14 has a incompleted section that the "Add a Instrumation Key to KeyValut" is missing detailed instructions.

**Comment by yoshioterada** [comment]
> @yoshioterada, Sorry my late response, I'd think that chapter-14 has a incompleted section that the "Add a Instrumation Key to KeyValut" is missing detailed instructions.

Thank you so much for your feedback @suzukiyuhsuke san. I will check and modify it.

**Comment by yoshioterada** [comment]
Dear @dingmeng-xue san

Yesterday, 
we had done the ASA-E workshop in Japan using this translated documents.
During the workshop, I got some feedbacks from the participants directlly.
And I immediately corrected any mistakes in wording or operational problems on the spot.

So now, all of the issue fixed and it is possible to merge it now.
So please merge this to the original.

File: azure-spring-apps-enterprise/ja-jp/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  * [作業 4 - アプリケーション内の機密情報の安全な読み込み](./azure-spring-apps-enterprise/README-jp.md#unit-4---securely-load-application-secrets)
+  * [作業 5 - エンドツーエンドのアプリケーション監視](./azure-spring-apps-enterprise/README-jp.md#unit-5---monitor-end-to-end)
+  * [作業 6 - リクエスト数に対するレート制限の設定](./azure-spring-apps-enterprise/README-jp.md#unit-6---set-request-rate-limits)
+  * [作業 7 - アイデアから生産までの自動化](./azure-spring-apps-enterprise/README-jp.md#unit-7---automate-from-idea-to-production)
+  * [作業 8 - フィットネス・ストアに AI を適用](./azure-spring-apps-enterprise/README-jp.md#unit-8---infuse-ai-into-fitness-store)
+
+## Tanzuアプリケーションプラットフォーム(TAP)へのアプリのデプロイ
+
+Tanzuアプリケーションプラットフォーム(TAP) を使用すると、多言語アプリを CNCF に準拠する Kubernetes クラスター上に、簡単に構築して実行できます。
+[tanzu-application-platform/README-jp.md](./tanzu-application-platform/README-jp.md) の記載内容に従い、アプリケーションを TAP にデプロイします。
```
Comment (dingmeng-xue) [inline azure-spring-apps-enterprise/ja-jp/README.md]: @yoshioterada , what's the correct link to this file? I cannot find it.

**Review by dingmeng-xue — APPROVED** [review]

**Pull request merged**